source/main.bmx

```diff
@@ -2428,8 +2443,21 @@ Type TSaveGame Extends TGameState
 				If st.availableScript
 					st.availableScript = TDatabaseLoader.ConvertOldAvailableScript(st.availableScript)
 				EndIf
+
+				migratedVariables :+ TDatabaseLoader.ConvertOldScriptExpression(st.templateVariables, migratedScriptExpression)
 			Next
-			'print "########## MIGRATED SCRIPT EXPRESSIONS IN SCRIPT TEMPLATES: " + migratedScriptExpressionCount +" #############"
+			'print "########## MIGRATED SCRIPT EXPRESSIONS IN SCRIPT TEMPLATES: " + migratedScriptExpressionCount +" templates, " + migratedVariables + " variables #############"
+
+
+			migratedScriptExpressionCount = 0
+			migratedVariables = 0
+			For local s:TScript = EachIn GetScriptCollection().entries.Values()
```

Comment:
Kann es sein, dass hier Einzelepisoden nicht mit konvertiert werden (nur Parent in der Collection)? Jedenfalls bleiben bei meinem Testspielstand %stationmaprandomcity%-Ausdrücke unkonvertiert.

Bei den Nachrichtenvorlagen gibt es dasselbe Problem vermutlich nicht, da die tatsächlich einzeln und nicht geschachtelt gehalten werden.